t/v2/key.t

```diff
@@ -116,7 +117,6 @@ checked val as expect: abc
 checked error msg as expect: Key not found
 
 
-
```

Comment:
we should keep this line.

https://github.com/openresty/openresty-devel-utils/blob/master/reindex

we use this tool to reformat test case.